posthog/api/error_tracking.py

```diff
@@ -0,0 +1,47 @@
+from django.db.models import QuerySet
+from rest_framework import serializers, viewsets
+
+from posthog.api.forbid_destroy_model import ForbidDestroyModel
+from posthog.api.routing import TeamAndOrgViewSetMixin
+from posthog.models.error_tracking import ErrorTrackingGroup
+
+
+def depluralize(string: str | None) -> str | None:
+    if not string:
+        return None
+
+    if string.endswith("ies"):
+        return string[:-3] + "y"
+    elif string.endswith("s"):
+        return string[:-1]
+    else:
+        return string
+
+
+class ErrorTrackingGroupSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ErrorTrackingGroup
+        fields = [
+            "assignee",
+        ]
+
+    def update(self, instance: ErrorTrackingGroup, validated_data: dict, **kwargs) -> ErrorTrackingGroup:
+        instance.assignee = validated_data["assignee"]
+        instance.save()
+        return instance
+
+
+class ErrorTrackingGroupViewSet(TeamAndOrgViewSetMixin, ForbidDestroyModel, viewsets.ModelViewSet):
+    scope_object = "error_tracking_group"
+    queryset = ErrorTrackingGroup.objects.all()
+    serializer_class = ErrorTrackingGroupSerializer
+
+    def safely_get_object(self, queryset) -> QuerySet:
```

Comment:
Do you need to add `.filter(team=self.team)` to querysets? I couldn't find if it was being appended or if not how we protect against accessing data from other teams

Reply:
yep, `safely_get_object` should be doing that for you

you can always write a test that creates errors in multiple teams and then assert that you only see the correct ones

Reply:
let's write some basic api tests 👍